governance/program/src/instruction.rs

```diff
@@ -410,10 +410,11 @@ pub enum GovernanceInstruction {
     ///
     ///   0. `[writable]` Realm account
     ///   1. `[signer]` Current Realm authority    
+    ///   2. `[]` New realm authority. Must be one of the realm governances when set
```

Comment:
I was considering it but it would make realm creation very inconvenient because you would have to create a proposal and vote on it to get governance (as the new realm authority) signature.

It would also be impossible to transfer realm authority from one governance to another because it's currently not possible to have proposal transaction to be be signed by more than one governance. 

The governance check ensures the new authority is set correctly and the only potential risk is if the realm members could not vote on proposals for the governance but in that case most likely the setup is already broken.

Reply:
> It would also be impossible to transfer realm authority from one governance to another because it's currently not possible to have proposal transaction to be be signed by more than one governance.

That's a perfect reason to not require the signature :laughing:  makes sense!